core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -184,6 +216,13 @@ public synchronized void reset() throws IOException {
     init(MasterInquireClient.Factory.create());
   }
 
+  /**
+   * @return the unique id of the context
+   */
+  public String getId() {
+    return mId;
```

Comment:
Is there any way to determine the ID of a process, how will users be able to map an ID to a process after seeing the metrics be abnormal?

Reply:
I considered this, but there does not seem a good approach commonly agreed on: https://stackoverflow.com/questions/35842/how-can-a-java-program-get-its-own-process-id?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

We could log the ID after it's created.

Reply:
Logging sounds good for now.